app/controllers/application_controller.rb

```diff
@@ -130,7 +130,7 @@ def query_search_options
                                                  :'query-not',
                                                  :'query-or',
                                                  :'query-quote')
-    query_search_params.inject({}) do |hash, kv|
+    query_search_params.to_h.inject({}) do |hash, kv|
```

Comment:
This change is unnecessary, as `query_search_params` are based on `permitted_params`, which now returns a hash.